examples/exact_values_example_1D.py

```diff
@@ -0,0 +1,50 @@
+from pykrige.ok import OrdinaryKriging
```

Comment:
Could you add a headline and a description for the example, like the other ones have?